src/components/navigation-bar/navigation-bar.spec.jsx

```diff
@@ -64,6 +64,14 @@ describe('Navigation Bar component', () => {
     expect(tabContent.props.title).toEqual('Custom Title 1');
   });
 
+  it('should have mountOnEnter prop set for every tab', () => {
+    const tabChildren = navigationBar.find(Tabs).props().children;
+    tabChildren.forEach((tab) => {
+      expect(tab.props.mountOnEnter).toEqual(true);
+    });
+    expect(tabChildren[1].props.mountOnEnter).toEqual(true);
```

Comment:
we should be able to remove this line now (it is redundant), since we are iterating on each tab now with the `forEach`

Reply:
Ahh I forgot removing this 👀

Reply:
I don't usually prefer loops inside of a test. They aren't explicit enough for me, but really just a preference.